tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -821,15 +821,34 @@ def test_positive_get_links(self):
         for group, path_pairs in api_paths.items():
             api_paths[group] = list(path_pairs.values())
 
-        if bz_bug_is_open(1166875):
-            # The server returns incorrect paths.
-            api_paths['docker_manifests'].append(u'/katello/api/docker_manifests')
-            api_paths['docker_manifests'].remove(u'/katello/api/compare')
-            api_paths['docker_tags'].append(u'/katello/api/docker_tags')
-            api_paths['docker_tags'].remove(u'/katello/api/compare')
-            api_paths['errata'].append(u'/katello/api/errata')
-            api_paths['errata'].append(u'/katello/api/errata/compare')
-            api_paths['errata'].remove(u'/katello/api/compare')
+            if bz_bug_is_open(1166875):
+                # The server returns incorrect paths.
+                API_PATHS['repositories'] = (
+                    u'/katello/api/repositories',
```

Comment:
This is not a bug , no need to add this in workaround , update the API path directly